.bazelrc

```diff
@@ -42,6 +42,8 @@ build --incompatible_strict_action_env
 
 # Add stack guards by default for better debugging, we disable them in release builds
 build --@seastar//:stack_guards=True
+# Debug mode by default. Disabled in release builds
+build --@seastar//:debug=True
```

Comment:
I am OK with this. We haven't fully defined what the buildmodes all mean, but fastbuild does NOT set NDEBUG, so I think this aligns well there.


Can we remove this option from the sanitizers then?